src/Core/functions.php

```diff
@@ -16,7 +16,14 @@
  */
 namespace Cake\Core;
 
+use Cake\I18n\Date;
+use Cake\I18n\DateTime;
+use DateTimeImmutable;
+use DateTimeInterface;
+use Exception;
+use IntlDateFormatter;
 use JsonException;
+use Opis\JsonSchema\Formats\DateTimeFormats;
```

Comment:
This isn't a package we have in our dependency list 🤔

Reply:
Wow, not sure where that came from, I was playing around with DateTime format constants and I guess my IDE pulled it in. Anyway, it's not in use so I will remove it. Good catch, thanks.